catalog/opensearch/catalog-opensearch-source/src/main/java/ddf/catalog/source/opensearch/OpenSearchParserImpl.java

```diff
@@ -33,96 +33,87 @@
 import ddf.catalog.impl.filter.SpatialFilter;
 import ddf.catalog.impl.filter.TemporalFilter;
 import ddf.catalog.operation.Query;
+import ddf.catalog.operation.QueryRequest;
 import ddf.catalog.source.UnsupportedQueryException;
+import ddf.catalog.source.opensearch.api.OpenSearchParser;
 import ddf.security.Subject;
 import ddf.security.assertion.SecurityAssertion;
 
-/**
- * Utility helper class that performs much of the translation logic used in CddaOpenSearchSite.
- */
-public final class OpenSearchSiteUtil {
+public class OpenSearchParserImpl implements OpenSearchParser {
 
-    // OpenSearch defined parameters
-    public static final String SEARCH_TERMS = "q";
+    private static final Logger LOGGER = LoggerFactory.getLogger(OpenSearchParserImpl.class);
 
-    // temporal
-    public static final String TIME_START = "dtstart";
+    // only for async searches
```

Comment:
oh, legacy comment that was moved around.  This enables pagination, which can be enabled by a client to do async.  I am not a fan of calling this async, which would imply to me the original call doesn't provide an answer, but a callback to get the answer from.